### PR TITLE
Fix/minor sass fixes

### PR DESCRIPTION
**Pull request by peterwilsoncc**
## Description

Various minor CSS fixes.

## How Has This Been Tested?

Visual checking of altered items.

## Types of changes

* Remove duplicate properties within a selector
* Remove unneeded units on zero values
* For `::selection` set `background-color` only as other background properties aren't supported

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style.
- [ ] My code has proper inline documentation.

### CODE REVIEW

File: blocks/library/shortcode/editor.scss
Comment (aduth) [inline blocks/library/shortcode/editor.scss]: This is a very obscure Flex bug affecting IE11: https://github.com/philipwalton/flexbugs#flexbug-4

File: blocks/color-palette/style.scss
Comment (aduth) [inline blocks/color-palette/style.scss]: cc @mtias , he likes this convention 😉 I'll leave it to him to defend.

**Review by aduth — CHANGES_REQUESTED** [review]
Flex bug change can't go through.

File: blocks/library/shortcode/editor.scss
Comment (peterwilsoncc) [reply blocks/library/shortcode/editor.scss]: Coolio, I'll revert and add a comment

File: blocks/color-palette/style.scss
Comment (peterwilsoncc) [reply blocks/color-palette/style.scss]: Yeah, I could see the logic but it's not used elsewhere in the project & it will throw an error in `stylelint-config-wordpress` once g7g is ready to merge into core.

As an aside, I'll set up a ticket to add `stylelint-config-wordpress` as a dependency to the project if it doesn't exist already.

Comment (ntwb) [reply blocks/color-palette/style.scss]: It doesn't have it currently, I attempted to do this in #1151, at the time the stylelint SCSS rules included in `stylelint-config-wordpress` needed to be updated, this remains the case today, an audit of the current rules included in the SCSS configuration of `stylelint-config-wordpress` is required.

File: blocks/library/shortcode/editor.scss
Comment (peterwilsoncc) [reply blocks/library/shortcode/editor.scss]: Fixed in a83363d608f010da2caef1c45a1d9f76dbabf2af.

**Comment by peterwilsoncc** [comment]
@aduth I've made the requested change and updated the PR against master.

**Review by ntwb — APPROVED** [review]
👍

**Review by aduth — APPROVED** [review]
Nice 👍

**Pull request merged**
